apps/angular/src/app/features/home/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,65 @@
+<!-- { results: [] } to keep table header displayed. -->
+@if ((animeList$ | async) || { results: [] }; as animeList) {
```

Comment:
To avoid this you can do something like this:
```suggestion
@if ({
  anime: animeList$ | async
}; as data) {
```